packaging/python/build_wheels.bash

```diff
@@ -97,6 +97,9 @@ build_wheel_osx() {
     echo " - Repairing..."
     delocate-wheel -w wheelhouse -v dist/*.whl  # we started clean, there's a single wheel
 
+    cd dist ; whl=`ls *.whl` ; cd ..
+    sh packaging/python/demo_update_libnrnmech.sh wheelhouse $whl
```

Comment:
@ferdonline : we should discuss this briefly today to see if you have better suggestions here.

Reply:
I looked at ```delocate-wheel -h``` and wonder if  the argument ``` -L LIB_SDIR   Subdirectory in packages to store copied libraries``` would server the purpose. i.e. ```-L .data/lib```. I'll try the experiment.

Reply:
Yes! It did work. Now, after installing the local wheel I see
```
$ otool -l "${NRNDEMO}release/x86_64/.libs/libnrnmech.so"
/Users/michaelhines/test37/lib/python3.7/site-packages/neuron/.data/share/nrn/demo/release/x86_64/.libs/libnrnmech.so:
...
Load command 10
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name @rpath/libnrniv.dylib (offset 24)
...
Load command 15
          cmd LC_RPATH
      cmdsize 48
         path @loader_path/../../../../../../lib (offset 12)
```
and
```
$ ls -al /Users/michaelhines/test37/lib/python3.7/site-packages/neuron/.dylibs
total 6688
drwxr-xr-x   9 michaelhines  staff      288 Oct 12 07:14 .
drwxr-xr-x  22 michaelhines  staff      704 Oct 12 07:14 ..
-rwxr-xr-x   1 michaelhines  staff   214656 Oct 12 07:14 libICE.6.dylib
-rwxr-xr-x   1 michaelhines  staff   106480 Oct 12 07:14 libSM.6.dylib
-rwxr-xr-x   1 michaelhines  staff  2404256 Oct 12 07:14 libX11.6.dylib
-rwxr-xr-x   1 michaelhines  staff    73888 Oct 12 07:14 libXau.6.dylib
-rwxr-xr-x   1 michaelhines  staff    87216 Oct 12 07:14 libXdmcp.6.dylib
-rwxr-xr-x   1 michaelhines  staff   172896 Oct 12 07:14 libXext.6.dylib
-rwxr-xr-x   1 michaelhines  staff   351280 Oct 12 07:14 libxcb.1.dylib
```
So no copy of libnrniv.dylib in ..../neuron/.dylibs

Reply:
But there is a puzzle. neurondemo continues to work when the ```-dll ..../.libs/libnrnmech.so``` is replaced by ```-dll ..../libnrnmech.dylib```, however
```
$ otool -l "${NRNDEMO}release/x86_64/libnrnmech.dylib"
/Users/michaelhines/test37/lib/python3.7/site-packages/neuron/.data/share/nrn/demo/release/x86_64/libnrnmech.dylib:
...
Load command 10
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name @rpath/libnrniv.dylib (offset 24)
...
Load command 15
          cmd LC_RPATH
      cmdsize 48
         path @loader_path/../../../../../../lib (offset 12)
```
But that is the same ```@loader_path``` with the 6 ```/..``` as for ```.../.libs/libnrnmech.so```